src/main/scala/monocly/OpticCan.scala

```diff
@@ -1,20 +1,22 @@
 package monocly
 
-//        GetMany
-//          /\
-//         /  \
-// GetOption  GetOneOrMore
-//         \  /
-//          \/
-//        GetOne
-trait GetMany
-trait GetOption extends GetMany
-trait GetOneOrMore extends GetMany
-trait GetOne extends GetOption with GetOneOrMore
+object OpticCan:
```

Comment:
I am on the fence, on one side, it is good not to pollute users with too many names, but it means either:
- the basic optic types are very verbose `POptic[OpticCan.Get & OpticCan.Modify, S, T, A, B]`
- or users need 2 imports `monocle._` and `monocle.Optican._`